pkg/term/term_windows.go

```diff
@@ -29,53 +31,93 @@ func StdStreams() (stdIn io.ReadCloser, stdOut, stdErr io.Writer) {
 		return os.Stdin, os.Stdout, os.Stderr
 	case os.Getenv("MSYSTEM") != "":
 		// MSYS (mingw) does not emulate ANSI well.
-		return winconsole.WinConsoleStreams()
+		return windows.ConsoleStreams()
 	default:
-		return winconsole.WinConsoleStreams()
+		return windows.ConsoleStreams()
 	}
 }
 
 // GetFdInfo returns file descriptor and bool indicating whether the file is a terminal.
 func GetFdInfo(in interface{}) (uintptr, bool) {
```

Comment:
the `Get` prefix is not idiomatic. I'd go with `FdInfo`, and the parameter should be `stream interface{}`.